### PR TITLE
test(e2e-tests): adds FF and glue for Cookies Disabled by default fea…

### PR DESCRIPTION
**Pull request by a6software**
…ture

## Ticket Number

AS-

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [ ] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/forms-web-app/src/app.js [hunk 1]
```diff
@@ -59,6 +60,30 @@ app
...
+
+const env = nunjucks.configure(viewPaths, nunjucksConfig);
+env.addFilter('appealSiteAddressToArray', appealSiteAddressToArray);
+env.addFilter('date', dateFilter);
+env.addFilter('formatBytes', fileSizeDisplayHelper);
+env.addFilter('filterByKey', filterByKey);
+env.addFilter('addKeyValuePair', addKeyValuePair);
+env.addGlobal('fileSizeLimits', config.fileUpload.pins);
+env.addGlobal('googleAnalyticsId', config.server.googleAnalyticsId);
+
```
Comment (a6software) [inline packages/forms-web-app/src/app.js:86]: change looks bigger than it is - had to move this to give access to `env` inside the `initialCookiesMiddleware` below. Surely there is a better way to do this? `res.locals` does not work and cost me several hours trying to make it work.

**Review by rcmsoftwaredevelopment — APPROVED** [review]

**Review by danWatkinson — COMMENTED** [review]
gets a tick from the cypress perspective

**Review by Fenisay — APPROVED** [review]
LGTM

**Comment by a6software** [comment]
ty

**Pull request merged**